### DISCUSSION TITLE
v4 Release date

### DISCUSSION THREAD

**Discussion by stephen-gabriele** [nativewind/nativewind#724] [General]
Do we have an idea of when v4 will be release? 

Excited for the ability to pass down tailwind classes in `className` for custom components

**Comment by pablo-k-neruda** [comment]
Something that I also have keen interest as Nativewind 4 + Expo router 3 is great combo for someone starting a new project now. Please let us know.

**Comment by marklawlor** [comment]
It will be released when I'm confident that people can migrate and have zero issues. Until then, your free to use it but there are no guarantees provided.

**Comment by corysimmons** [comment]
It looks like there is a TODO here: https://github.com/marklawlor/nativewind/blob/main/v4-TODO.md

If anyone wants to contribute, you could probably reference the existing tests in the v4 branch and start PR'ing a few tests here and there. A lot of this is already set up and just needs to be copied/pasted and filled out with more examples of code developers might actually use.

- https://github.com/marklawlor/nativewind/tree/main/packages/nativewind/src/__tests__
- https://github.com/marklawlor/nativewind/tree/main/packages/react-native-css-interop/src/__tests__

Specifically, the TODO mentions needing responsive design and SVG tests:

- https://github.com/marklawlor/nativewind/blob/main/packages/nativewind/src/__tests__/responsive-design.tsx
- https://github.com/marklawlor/nativewind/blob/main/packages/nativewind/src/__tests__/svg.tsx (these look easy to contribute to)
